### DISCUSSION TITLE
Merged rows

### DISCUSSION THREAD

**Discussion by wxng** [thombashi/pytablewriter#40] [Q&A] [answered]
May I know if the library supports creating tables with merged rows?

E.g. ([source](https://stackoverflow.com/questions/46621765/can-i-merge-table-rows-in-markdown)):
![image](https://user-images.githubusercontent.com/11936133/120760162-618dd600-c546-11eb-84eb-435c993ed39d.png)

**Comment by thombashi** [comment, marked as answer]
Unfortunately, not supported for now.

I made issue #41
